### PR TITLE
[BLR-2022] Add Bengaluru 2022

### PR DESCRIPTION
**Pull request by vjdhama**
Adding Bengaluru 2022.

### CODE REVIEW

File: data/events/2022-bengaluru.yml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  - name: "Praveen Shukla"
+    twitter: "_praveenshukla"
+  - name: "Vijay Dhama"
+    twitter: "vjdhama"
+  - name: "Saiyam Pathak"
+    github: "SaiyamPathak"
+  - name: "Leena S N"
+    twitter: "leenasn"
+
+organizer_email: "" # Put your organizer email address here
```
Comment (bridgetkromhout) [inline data/events/2022-bengaluru.yml:59]: This cannot be blank - you need your cityname@devopsdays.org set up here.

**Comment by vjdhama** [comment]
@bridgetkromhout Updated the PR with the changes.

We needed help with Slack access(because we lost access to old emails) and updating the organizer email group, that's why I didn't add that before. We did send an email(titled "Help with Slack access") to info.devopsdays.org, but got no response for the same even after followups. 

We'd be grateful if you can also help with that.

Also, do I need to squash the commits?

File: data/events/2022-bengaluru.yml (hunk 1 above)
Comment (vjdhama) [reply data/events/2022-bengaluru.yml]: Updated.

File: data/events/2022-bengaluru.yml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+  - name: "Praveen Shukla"
+    twitter: "_praveenshukla"
+  - name: "Vijay Dhama"
+    twitter: "vjdhama"
+  - name: "Saiyam Pathak"
+    github: "SaiyamPathak"
+  - name: "Leena S N"
+    twitter: "leenasn"
+  - name: "Ashwini Manoj"
+    twitter: "_scarletfire_"
```
Comment (phrawzty) [inline data/events/2022-bengaluru.yml:59]: Please send the names, email addresses, and organisations these individuals represent to `info@devopsdays.org`. From there we can sort out the mailing list and the Slack invitations. 👍

Comment (vjdhama) [reply data/events/2022-bengaluru.yml:59]: @phrawzty I have send those details over email.

**Comment by vjdhama** [comment]
@yvovandoorn Could you please let me know what I am missing in the PR?

**Pull request merged**